packages/acter_notifify/lib/util.dart

```diff
@@ -30,7 +30,7 @@ final useLocal = Platform.isAndroid ||
 final usePush = Platform.isAndroid || Platform.isIOS;
 
 Future<int> notificationsCount() async {
-  if (Platform.isLinux) return 0; // not supported
+  if (Platform.isLinux || Platform.isWindows) return 0; // not supported
```

Comment:
```suggestion
  if (!supportsLocalBadgeCount) return 0; // not supported
```

Reply:
I don't know why `isMacOS` was missed in original condition.

Reply:
You already defined `useLocal` flag.
I will use it.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2650/commits/54631688bed25794dbc54a234a9fe669b2822939